ci-operator/step-registry/crc/e2e/test/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- praveenkumar
```

Comment:
Any chance there is more than one owner for this and the remaining owners files?

Reply:
@alvaroaleman not right now but will add as we go.

Reply:
Who will review your changes to this step?

Reply:
@petr-muller I need to add on of the team member